src/AddressRegistry/StreetName/Events/AddressWasProposedBecauseOfReaddressing.cs

```diff
@@ -0,0 +1,122 @@
+namespace AddressRegistry.StreetName.Events
+{
+    using System.Linq;
+    using Be.Vlaanderen.Basisregisters.EventHandling;
+    using Be.Vlaanderen.Basisregisters.GrAr.Common;
+    using Be.Vlaanderen.Basisregisters.GrAr.Provenance;
+    using Newtonsoft.Json;
+
+    [EventTags(EventTag.For.Sync, EventTag.For.Edit)]
+    [EventName(EventName)]
+    [EventDescription("Het adres werd voorgesteld door een heradressering.")]
+    public class AddressWasProposedBecauseOfReaddressing : IStreetNameEvent, IHasAddressPersistentLocalId
```

Comment:
Do you guys prefer BecauseOfReaddress or BecauseOfReaddressing or ...?

Reply:
Readdress 👍